dbt/parser.py

```diff
@@ -91,11 +91,16 @@ def process_refs(flat_graph, current_project):
                 current_project,
                 node.get('package_name'))
 
+            node_is_test = (node.get('resource_type') == NodeType.Test)
             if target_model is None:
                 dbt.exceptions.ref_target_not_found(
                     node,
                     target_model_name,
-                    target_model_package)
+                    target_model_package,
+                    warn=node_is_test)
+
+                if node_is_test:
+                    continue
```

Comment:
this doesn't seem right to me... these `dbt.exceptions` calls should raise. i'd pull the message out of `ref_target_not_found` and then make this more explicit:

```
if target_model is None:
    if node.get('resource_type') == NodeType.Test:
        logger.info(warning)
        continue
    else:
        dbt.exceptions.ref_target_not_found( ... )
```